packages/deprecation-crawler/src/lib/tasks/ensure-fotmatters.ts

```diff
@@ -0,0 +1,36 @@
+import { EOL } from "os";
+import { builtInFormatter } from "../output-formatters";
+import { CRAWLER_CONFIG_PATH } from "../constants";
+import { CrawlConfig, Deprecation } from "../models";
+
+export function ensureFormatter(config: CrawlConfig): [string, (config: CrawlConfig,
+  rawDeprecations: Deprecation[]
+) => Promise<void>][] {
+  if(config.outputFormatters.length <= 0) {
+    throw new Error(`No formatter registered! ${EOL}
```

Comment:
I don't think this should throw in the future.
It could be possible that a user has its own formatter.